stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/composer/MessageComposerContext.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2014-2022 Stream.io Inc. All rights reserved.
+ *
+ * Licensed under the Stream License;
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://github.com/GetStream/stream-chat-android/blob/main/LICENSE
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.getstream.chat.android.ui.message.composer
+
```

Comment:
An object that is propagated to each content view in each content slot. It can hold more data, maybe even listeners. Don't want anything else to be added to the content view lifecycle, just 2 methods: `attachContext` when the content view is mounted to the hierarchy and we need to do some initialization like styling, and `renderState` to render the current state.

```
public interface MessageComposerContent {

    public fun attachContext(messageComposerContext: MessageComposerContext)

    public fun renderState(state: MessageComposerState)
}
```